### DISCUSSION TITLE
配置文件里任务已都设置为true，但是没有脚本日志输出

### DISCUSSION THREAD

**Discussion by SHOPPER520** [0-8-4/miui-auto-tasks#270] [问与答｜Q&A]
08:30:02 - INFO  | system_info.print_info:13 - MIUI-AUTO-TASK v1.7.5
08:30:02 - INFO  | system_info.print_info:14 - ---------- 系统信息 -------------
08:30:02 - INFO  | system_info.system_info:25 - 操作系统平台: Linux-6.1.22-x86_64-with
08:30:02 - INFO  | system_info.system_info:26 - 操作系统版本: #1 SMP PREEMPT_DYNAMIC Thu Jan 11 03:42:31 UTC 2024
08:30:02 - INFO  | system_info.system_info:27 - 操作系统名称: Linux
08:30:02 - INFO  | system_info.system_info:28 - 操作系统位元: ('64bit', '')
08:30:02 - INFO  | system_info.system_info:29 - 操作系统类型: x86_64
08:30:02 - INFO  | system_info.system_info:30 - 处理器信息: 
08:30:02 - INFO  | system_info.system_info:31 - Python 版本: 3.11.8 ('main', 'Feb  7 2024 23:34:27')
08:30:02 - INFO  | system_info.print_info:16 - ---------- 项目信息 -------------
08:30:02 - INFO  | system_info.print_info:17 - 这是一个免费且开源的项目，如果你是付费购买获得请务必退款
08:30:02 - INFO  | system_info.print_info:18 - 项目地址：https://github.com/0-8-4/miui-auto-tasks
08:30:02 - INFO  | system_info.print_info:19 - 欢迎 star，感谢所有项目贡献者，已经提交issues的人，帮助项目发展的人
08:30:02 - INFO  | system_info.print_info:20 - ---------- 脚本日志 -------------
08:30:03 - SUCCESS | login.login:83 - 小米账号登录成功

## 执行结束...   耗时 4 秒　　　　　

配置文件里任务已都设置为true，但是没有脚本日志输出

**Comment by MeepoMaster** [comment]
一样

**Reply by MeepoMaster** [reply]
accounts:
-   uid: 'uid'
    password: password
    cookies:
        passToken: passToken
    login_user_agent: ''
    user_agent: user_agent
    device: ''
    device_model: ''
    CheckIn: true
    BrowseUserPage: true
    BrowsePost: true
    ThumbUp: true
    BrowseSpecialPage: true
    BoardFollow: true
    CarrotPull: true

------------------------------------------------------------------------------------------------------------------------------

## 开始执行... 2024-04-07 08:30:00

08:30:01 - INFO  | system_info.print_info:13 - MIUI-AUTO-TASK v1.7.5
08:30:01 - INFO  | system_info.print_info:14 - ---------- 系统信息 -------------
08:30:01 - INFO  | system_info.system_info:25 - 操作系统平台: Linux-6.5.0-1019-oracle-aarch64-with
08:30:01 - INFO  | system_info.system_info:26 - 操作系统版本: #19~22.04.1-Ubuntu SMP Tue Mar 12 23:02:13 UTC 2024
08:30:01 - INFO  | system_info.system_info:27 - 操作系统名称: Linux
08:30:01 - INFO  | system_info.system_info:28 - 操作系统位元: ('64bit', '')
08:30:01 - INFO  | system_info.system_info:29 - 操作系统类型: aarch64
08:30:01 - INFO  | system_info.system_info:30 - 处理器信息: 
08:30:01 - INFO  | system_info.system_info:31 - Python 版本: 3.11.8 ('main', 'Mar 16 2024 09:26:53')
08:30:01 - INFO  | system_info.print_info:16 - ---------- 项目信息 -------------
08:30:01 - INFO  | system_info.print_info:17 - 这是一个免费且开源的项目，如果你是付费购买获得请务必退款
08:30:01 - INFO  | system_info.print_info:18 - 项目地址：https://github.com/0-8-4/miui-auto-tasks
08:30:01 - INFO  | system_info.print_info:19 - 欢迎 star，感谢所有项目贡献者，已经提交issues的人，帮助项目发展的人
08:30:01 - INFO  | system_info.print_info:20 - ---------- 脚本日志 -------------
08:30:06 - SUCCESS | login.login:83 - 小米账号登录成功

------------------------------------------------------------------------------------------------------------------------------

配置文件里全部都是true，显示登录成功，但是仅仅只是登录成功

**Reply by 0-8-4** [reply]
我这边没有浮现 这个问题喔

我也是全true的

**Reply by koolcandy** [reply]
没有遇到

**Reply by BA7JCM** [reply]
login_user_agent，device，device_model，这三个参数填了没，尤其是第一个

**Comment by BA7JCM** [comment]
仔细检查配置文件，有没有空白项，尤其是两处UA

**Reply by MeepoMaster** [reply]
都没，看到Wiki里没有，就不知道咋填

**Comment by SHOPPER520** [comment]
果然 login_user_agent这个参数必须填写！填写之后日志可以跑起来！但我的好像人机验证过不了
